data/translations/en/state-data.json

```diff
@@ -864,10 +964,11 @@
     "more_info_link": "https://www.vivote.gov/voters/register-vote",
     "more_info_link_translated": "true",
     "more_info_link_language_selection": "",
-    "ip_deadline": "30 days before Election Day",
+    "mail_postmarked_deadline": "Must be postmarked by Sunday, October 09, 2022",
```

Comment:
no `default_` value for postmarked deadline. Is this new?

Reply:
Yes, it is new :)

Reply:
confirmed